notebooks/00_quick_start/xdeepfm_movielens.ipynb

```diff
@@ -0,0 +1,1541 @@
+{
```

Comment:
<p>what I was saying in the other comments is removing the ymal file and add here a dictionary of the parameters we want to add. I think it is easier for the user to understand. Maybe we can explain some of them</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/861/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>